airgun/views/containerimages.py

```diff
@@ -0,0 +1,63 @@
+from widgetastic.widget import Text, View
+from widgetastic_patternfly5.ouia import Button as PF5OUIAButton
+
+from airgun.views.common import BaseLoggedInView, PF4Search
+from airgun.widgets import CompoundExpandableTable  # Import the new widget
+
+
+class ContainerImagesView(BaseLoggedInView):
+    title = Text('.//h1[@data-ouia-component-id="container-images-title"]')
+    searchbox = PF4Search()
```

Comment:
We really should rename that `PF4Search()` to just `PFSearch()`, as it does not seem like it's fixed to the PF version.
Not a scope of this PR, though.